simulator/tabletop.py

```diff
@@ -74,3 +89,26 @@ def start_app_on_all(self, app_type):
   def run(self):
     """ Runs the tabletop simulation indefinitely. """
     self.__canvas.wait_for_events()
+  
+  def draw_grid(self):
+    """ Creates Line objects for grid, returns the list of line objects in grid."""
+    grid = []
+    window_x, window_y = self.__canvas.get_window_size()
+    i = 0 
+    while i < window_x:
+      line = Line(self.__canvas,( i, 0), (i, window_y))
+      grid.append(line)
+      i += 200
```

Comment:
I believe we have a ```CUBE_SIZE``` constant that should be used here instead of hardcoding it.